app/models/source_records.py

```diff
@@ -35,3 +35,9 @@ class SourceRecord(BaseModel):
     contributions: List[SourceContribution] = []
 
     issue: Optional[SourceIssue] = None
+
+    @model_validator(mode="after")
+    def validate_titles(self):
+        """Validate that the title field is not empty"""
+        if not self.titles:
+            raise ValueError("Source Record titles cannot be empty")
```

Comment:
- What is the the expected result of this raised exception
- To operate on a sigle field, use field_validator instead of model_validator
- Field has a min_length attribute that can be used alternatively